frontends/web/src/routes/settings/manage-accounts.tsx

```diff
@@ -226,6 +253,25 @@ class ManageAccounts extends Component<Props, State> {
                                 </div>
                             </div>
                         ) : null }
+                        { editAccountCode ? (
+                            <Dialog
+                                onClose={() => this.setState({ editAccountCode: undefined, editAccountNewName: '' })}
```

Comment:
why name to `''` and not to `undefined`?

Reply:
this will be passed to the input element, and it should do `input.value = ''` and not `input.value = undefined`.